shim/linux/src/entry.c

```diff
@@ -192,7 +193,9 @@ dispatch_system_kvm_create_vm(void)
 static long
 dispatch_system_kvm_get_api_version(void)
```

Comment:
Please check that the ioctl_args are empty before calling this function (after the `case KVM_GET_API_VERSION: {` line) and fail with `-EINVAL` otherwise. This should be done for every ioctl that don't take any argument in order to have the same behavior as KVM.